src/main/thrift/octopus.thrift

```diff
@@ -0,0 +1,62 @@
+namespace scala com.gu.flexibleoctopus.model.thrift
+
+/** date times are reprsented as i64 - epoch millis */
+typedef i64 DateTime
+
+/** 
+* A single story bundle that maps directly to a piece of content in Composer 
+*/
+struct StoryBundle {
+
+    1: required i64 id;
+
+    2: required string composerId;
+
+    3: required string pubCode;
+
+    4: required string sectionCode;
+
+    5: required Article article;
+
+}
+
+/**
+* Represents a single article of a story bundle as generated by inCopy, 
+* such as the headline, standfirst, or body text
+*/
+struct Article {
+
+    // id for the article
+    1: required i64 id;
+
+    // id for the book section
+    2: required i64 bookSectionId;
+
+    // indicates whether the article is for web and/or print pulication
+    3: required string forPublication;
+
+    // current user that has the article checked out
+    4: required string inUseBy; 
+
+    // whether the article is currently checked out by a user
+    5: required bool isCheckedOut;
+
+    // when the last version of the article was saved
+    6: required DateTime lastModified;
+
+    // current legal status of the article
+    7: required string lawyered;
```

Comment:
should this be a boolean?

Reply:
We have three known values: C, P, R. It might make sense to make this an enum, and since I don't know the relevance of those values, I think also including a comment to explain their meaning / naming them more explicitly would make sense.

Reply:
P = "Pending" i.e. it hasn't yet been cleared by a lawyer. It's the default for all articles.
C = "Cleared" i.e. a lawyer has marked this as OK
R = "pRiority" i.e. a subeditor has pushed this to the top of the queue in the Lawyers' Octopus application (which they use to review Octopus content(

Reply:
If this is for "lawyered" then it breaks down as follows:

C = Cleared (a lawyer has read print version and is happy)
P = Pending (it is waiting to be read by a lawyer)
R = Priority (oy! Lawyer look at this very soon please)
N = Not Applicable (Don't bother reading this)